pom.xml

```diff
@@ -40,6 +40,35 @@
                         <Private-Package>org.orbisgis.mapuce.*</Private-Package>
                         <Bundle-Category>OrbisGIS,WPS,script,WpsScriptPlugin</Bundle-Category>
                         <Bundle-Icon>pluginIcon.png;size=32</Bundle-Icon>
+                        <Embed-Dependency>
+                            weka-stable;scope=compile;inline=true,
+                            java-cup-11b-runtime;scope=compile;inline=true,
+                            java-cup-11b;scope=compile;inline=true,
+                            mtj;scope=compile;inline=true,
+                            bounce;scope=compile;inline=true,
+                            commons-compress;scope=compile;inline=true,
+                            arpack_combined_all;scope=compile;inline=true,
+                            netlib-java;scope=compile;inline=true,
+                            netlib-native_ref-linux-i686;scope=compile;inline=true,
+                            netlib-native_ref-linux-x86_64;scope=compile;inline=true,
+                            netlib-native_ref-osx-x86_64;scope=compile;inline=true,
+                            netlib-native_ref-win-i686;scope=compile;inline=true,
+                            netlib-native_ref-win-x86_64;scope=compile;inline=true,
+                            netlib-native_ref-linux-armhf;scope=compile;inline=true,
+                            netlib-native_system-linux-i686;scope=compile;inline=true,
+                            netlib-native_system-linux-armhf;scope=compile;inline=true,
+                            netlib-native_system-linux-x86_64;scope=compile;inline=true,
+                            netlib-native_system-osx-x86_64;scope=compile;inline=true,
+                            netlib-native_system-win-i686;scope=compile;inline=true,
+                            netlib-native_system-win-x86_64;scope=compile;inline=true,
+                            netlib-native_ref;scope=compile;inline=true,
+                            netlib-native_system;scope=compile;inline=true,
+                            core;scope=compile;inline=true,
+                            jniloader;scope=compile;inline=true,
+                            ant-launcher;scope=compile;inline=true,
+                            ant;scope=compile;inline=true,
+                            xz;scope=compile;inline=true
+                        </Embed-Dependency>
```

Comment:
It seems that the change has been done in is github repo, in the branch dev : [dev/pom.xml](https://github.com/Melviyn/MApUCE_tools/commit/dc4b3455d327405b98bcc9111c17e56b54ae8469)